catalog/files/api/src/main/java/org/projectnessie/catalog/files/api/ObjectIO.java

```diff
@@ -29,4 +32,22 @@ public interface ObjectIO {
   OutputStream writeObject(StorageUri uri) throws IOException;
 
   void deleteObjects(List<StorageUri> uris) throws IOException;
+
+  void icebergWarehouseConfig(
+      StorageUri warehouse,
+      BiConsumer<String, String> defaultConfig,
+      BiConsumer<String, String> configOverride);
+
+  void icebergTableConfig(
```

Comment:
`ObjectIO` is kind of Nessie's internal access channel to storage... Why overload it with Iceberg FileIO stuff? How about adding a separate interface (e.g. `IcebergIOConfigurer`) for the new methods?